client/dashboard/components/responsive-menu/index.tsx

```diff
@@ -44,6 +159,62 @@ function ResponsiveMenu( {
 				} ) }
 			</Menu>
 		);
+
+		return (
+			<HStack spacing={ 3 }>
+				{ prefix }
+				<div
+					style={ {
+						position: 'relative',
+						width: '100%',
+						display: 'flex',
+						overflow: 'visible',
+					} }
+				>
+					<ScrollButton
+						style={ {
+							left: '-2px',
+							paddingRight: '30px',
+							background:
+								'linear-gradient(90deg, var(--dashboard__background-color) 30%, transparent 100%)',
+							opacity: showLeftButton ? 1 : 0,
+							pointerEvents: showLeftButton ? 'auto' : 'none',
+						} }
+						onClick={ () => bumpScrollX( true ) }
+					>
+						<Icon icon={ chevronLeft } />
+					</ScrollButton>
+
+					<div
+						ref={ containerRef }
+						style={ {
+							overflowX: 'scroll',
+							scrollbarWidth: 'none',
+							msOverflowStyle: 'none',
```

Comment:
Do we need to handle chrome "::-webkit-scrollbar"?

Reply:
I think so. Fixed: a42ca884d69

Though it is impossible to do that in inline styles 😄